### ISSUE TITLE
upgrade to Qt 4.8.7_1 breaks bottle for 10.9.5

### ISSUE THREAD

**Issue by cbertelli** [OSGeo/homebrew-osgeo4mac#114]
Hello,
nothing preventing usage, but icons are not displayed. At least this is what happens to me.

**Comment by esagara** [comment]
I am seeing the same issue. It does appear to be a Qt related issue from what I can see when running the wrapper script.
```
Warning: loading of qgis translation failed [/usr/local/Cellar/qgis-28/2.8.3/QGIS.app/Contents/MacOS/../Resources/i18n//qgis_en_US]
Warning: loading of qt translation failed [/usr/local/Cellar/qt/4.8.7_1/translations/qt_en_US]
Warning: QGraphicsScene::addItem: item has already been added to this scene
2015-10-14 00:40:39.727 QGIS[64553:507] modalSession has been exited prematurely - check for a reentrant call to endModalSession:
```

**Comment by russellgoldenberg** [comment]
Getting same error on 10.10.5

**Comment by cbertelli** [comment]
That's what I did.
brew untap osgeo/osgeo4mac
brew tap christianurich/osgeo4mac
because Christian Urich updated the formula from 2.8 to 2.10.
Then I did
brew install qgis
to obtain a new perfectly working version 2.10.
That's it. Now I'm waiting for version 2.12.
c


On Mon, Nov 2, 2015 at 10:29 PM, Russell Goldenberg <
notifications@github.com> wrote:

> Getting same error on 10.10.5
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OSGeo/homebrew-osgeo4mac/issues/114#issuecomment-153163181>
> .
>



-- 
--------------------------------------------------------------------------
Carlo A. Bertelli
   Charta servizi e sistemi per il territorio e la storia ambientale srl
          Dipendenze del palazzo Doria,
          vc. alla Chiesa della Maddalena 9/2 16124      Genova (Italy)
          tel./fax +39(0)10 2475439  +39 0108566195  gsm:+39 393 1590711
   e-mail: bertelli@chartasrl.eu      http://www.chartasrl.eu
--------------------------------------------------------------------------
